lib/exercism/user_exercise.rb

```diff
@@ -51,6 +51,18 @@ def unarchive!
     update_attributes(archived: false)
   end
 
+  def help_requested?
+    help_requested
+  end
+
+  def request_for_help!(user)
```

Comment:
Why is `user` a useful parameter here?
The exercise already belongs to a user, only that user should be able to access the object and call request_for_help! on it.